Config with default template ee0c5463

mauritsvanrees opened the pull request.

mister-roboto commented:
@mauritsvanrees thanks for creating this Pull Request and helping to improve Plone!

TL;DR: Finish pushing changes, pass all other checks, then paste a comment:
```
@jenkins-plone-org please run jobs
```

To ensure that these changes do not break other parts of Plone, the Plone test suite matrix needs to pass, but it takes 30-60 min.  Other CI checks are usually much faster and the Plone Jenkins resources are limited, so when done pushing changes and all other checks pass either [start all Jenkins PR jobs yourself](https://jenkinsploneorg.readthedocs.io/en/latest/run-pull-request-jobs.html#run-a-pull-request-job), or simply add the comment above in this PR to start all the jobs automatically.

Happy hacking!

mauritsvanrees commented:
Note that this adds a dependency on CMFPlone. Do we want this at this point in time?

@jenkins-plone-org please run jobs

jensens commented on setup.py:
```diff
@@ -36,20 +36,47 @@
-        "plone.app.z3cform",
+        "plone.protect",
+        "plone.registry",
+        "Products.CMFCore",
```
I do not think it is a good idea to introduce circular dependencies.

jensens commented on setup.py:
```diff
@@ -36,20 +36,47 @@
+        "setuptools",
         "transaction",
+        "z3c.form",
+        "ZODB",
```
this should come with Zope as transitive dependency. Same for some others.

jensens reviewed (changes requested):
Dependency declaration in setup.py does not take transitives into account. plone.base is not in there. Isn't it so, that  the script should detect this on its own? Seems there is something broken.

mauritsvanrees commented:
> Dependency declaration in setup.py does not take transitives into account. plone.base is not in there. Isn't it so, that the script should detect this on its own? Seems there is something broken.

Initially I left all existing dependencies there, and only added missing ones reported by dependencychecker, and removed reported unneeded ones.  That does not take transitive dependencies into account.  It is apparently best to start with a completely empty list, and let dependencychecker do its job.  I have done so now.
And I have removed CMFPlone.
